<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrobblingSettings</class>
 <widget class="QWidget" name="ScrobblingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="BuddyLabel" name="label_2">
     <property name="text">
      <string>Username:</string>
     </property>
     <property name="buddy">
      <cstring>user</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="LineEdit" name="user"/>
   </item>
   <item row="1" column="0">
    <widget class="BuddyLabel" name="label_3">
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="buddy">
      <cstring>pass</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="LineEdit" name="pass">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="BuddyLabel" name="label_5">
     <property name="text">
      <string>Status:</string>
     </property>
     <property name="buddy">
      <cstring>loginButton</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="loginStatusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="MessageWidget" name="messageWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="enableScrobbling">
     <property name="text">
      <string>Scrobble tracks to last.fm</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>support/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>support/buddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>MessageWidget</class>
   <extends>QFrame</extends>
   <header>support/messagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
